Server/src/server/gameCenter/models/game/Game.java

```diff
@@ -961,7 +991,7 @@ private TargetData detectOnDeathTarget(Spell spell, Cell cardCell, Cell clickCel
 
         player = (getCurrentTurnPlayer().getPlayerNumber() == playerNumber) ? getCurrentTurnPlayer()
             : getOtherTurnPlayer();
-
+        System.out.println("we should be here");
```

Comment:
what does this mean?

Reply:
That's me being sloppy in removing print statements again